src/app/item-popup/EnergyMeter.tsx

```diff
@@ -51,7 +51,7 @@ export default function EnergyMeter({
   const minCapacity = previewEnergyType === energyType ? energyCapacity : 1;
 
   // layer in possible total slots, then earned slots, then currently used slots
-  const meterIncrements = Array(10)
+  const meterIncrements: string[] = Array(10)
```

Comment:
kind of prefer this?
`const meterIncrements = Array<string>(10)`
defines it closer to the source than declaring the intention of the destination variable.
this way even the `fill` operations are typechecked

Reply:
Ah yeah that's better.